extensions/machine-learning-services/src/test/packageManagement/packageManagementService.test.ts

```diff
@@ -23,74 +23,64 @@ function createContext(): TestContext {
 	};
 }
 
-describe('Server Config Manager', () => {
+describe('Package Management Service', () => {
 	it('openDocuments should open document in browser successfully', async function (): Promise<void> {
 		const context = createContext();
 		context.apiWrapper.setup(x => x.openExternal(TypeMoq.It.isAny())).returns(() => Promise.resolve(true));
-		let serverConfigManager = new ServerConfigManager(context.apiWrapper.object, context.queryRunner.object);
+		let serverConfigManager = new PackageManagementService(context.apiWrapper.object, context.queryRunner.object);
 		should.equal(await serverConfigManager.openDocuments(), true);
 	});
 
 	it('openOdbcDriverDocuments should open document in browser successfully', async function (): Promise<void> {
 		const context = createContext();
 		context.apiWrapper.setup(x => x.openExternal(TypeMoq.It.isAny())).returns(() => Promise.resolve(true));
-		let serverConfigManager = new ServerConfigManager(context.apiWrapper.object, context.queryRunner.object);
+		let serverConfigManager = new PackageManagementService(context.apiWrapper.object, context.queryRunner.object);
 		should.equal(await serverConfigManager.openOdbcDriverDocuments(), true);
 	});
 
 	it('openInstallDocuments should open document in browser successfully', async function (): Promise<void> {
 		const context = createContext();
 		context.apiWrapper.setup(x => x.openExternal(TypeMoq.It.isAny())).returns(() => Promise.resolve(true));
-		let serverConfigManager = new ServerConfigManager(context.apiWrapper.object, context.queryRunner.object);
+		let serverConfigManager = new PackageManagementService(context.apiWrapper.object, context.queryRunner.object);
 		should.equal(await serverConfigManager.openInstallDocuments(), true);
 	});
 
 	it('isMachineLearningServiceEnabled should return true if external script is enabled', async function (): Promise<void> {
```

Comment:
nit: test for if not installed

Reply:
this seems s=correct. not sure what you mean

---
In reply to: [393910111](https://github.com/microsoft/azuredatastudio/pull/9519#discussion_r393910111) [](ancestors = 393910111)

Reply:
oh I meant - we are testing these functions return yes if ext script is enables and R is installed - but no negative test cases for when its not enabled/installed. But leave it to u to add or not...

Reply:
ah got it. sure will add that. maybe in another PR which I'm adding all the tests

---
In reply to: [395375016](https://github.com/microsoft/azuredatastudio/pull/9519#discussion_r395375016) [](ancestors = 395375016)